api/src/main/java/org/apache/iceberg/actions/RewriteManifests.java

```diff
@@ -44,6 +45,20 @@ public interface RewriteManifests
    */
   RewriteManifests rewriteIf(Predicate<ManifestFile> predicate);
 
+  /**
+   * Rewrite manifests in a given order, based on partition transforms
+   *
+   * <p>If not set, manifests will be rewritten in the order of the transforms in the table's
+   * current partition spec.
+   *
+   * @param partitionSortOrder a list of partition field transform names
+   * @return this for method chaining
+   */
+  default RewriteManifests sort(List<String> partitionSortOrder) {
```

Comment:
I think this should be better called `partitionFieldSortOrder`? because the inputs are not the actual partitions, but the partition fields.

Reply:
Works for me, apologies I'm still struggling with the naming conventions a bit.